sdk/keyvault/azure-security-keyvault-keys/README.md

```diff
@@ -326,20 +326,6 @@ for (auto keys = client.GetPropertiesOfKeys(); keys.HasPage(); keys.MoveToNextPa
     }
 ```
 
-<!-- ### Encrypt and Decrypt
-
-This example creates a `CryptographyClient` and uses it to encrypt and decrypt with a key in Azure Key Vault.
-
-```cpp
-byte[] plaintext = Encoding.UTF8.GetBytes("A single block of plaintext");
-
-// encrypt the data using the algorithm RSAOAEP
-EncryptResult encryptResult = cryptoClient.Encrypt(EncryptionAlgorithm.RsaOaep, plaintext);
-
-// decrypt the encrypted data.
-DecryptResult decryptResult = cryptoClient.Decrypt(EncryptionAlgorithm.RsaOaep, encryptResult.Ciphertext);
-``` -->
-
```

Comment:
Does this sample not work for some reason? Is this feature/functionality not relevant to users? If we are removing it, just remove this particular section from the headings above, rather than all of them.

Reply:
this was commented out, i removed it as it causes iossued in the rendered HTML 
![image](https://user-images.githubusercontent.com/50641385/148479245-99529acc-e496-4901-accb-e5df6bbc5ba4.png)